src/getting-started-admin.handlebars

```diff
@@ -83,10 +83,11 @@
                                         <h3>With a command line</h3>
                                         <p>Alternatively, you can create the client id and secret with a single command line directly on the PIM server.</p>
                                         <h4>With the 2.0 version</h4>
-                                        <pre class="hljs"><code>php bin/console pim:oauth-server:create-client \
+                                        <pre class="hljs"><code>php bin/console pim:oauth-server:create-client my-customer-account \
```

Comment:
add `--label my-customer-account` in the command of 1.7 also please (as the result display the label and is the same in 2.0 and 1.7)